### DISCUSSION TITLE
Hisense AEH-W4E1

### DISCUSSION THREAD

**Discussion by kisjeth** [indomus/forum#1006] [HUB: Home Assistant]
Buongiorno. C'è modo di integrare i climatizzatori Hisense con modulo AEH-W4E1 ad Home Assistant?
L'integrazione nativa è solo per il modulo W4A1, non c'è altro modo per configurare il dispositivo?

**Comment by essegi65** [comment]
Io sono riuscito la scorsa settimana ad integrare il mio nuovo condizionatore  (stesso modulo wifi del tuo) con hisense air conditioners [qui](https://github.com/deiger/AirCon). al momento ho inserito in lovelace il termostato ma non ho avuto il tempo di fare delle prove complete solo piccole accensioni di qualche minuto.

**Reply by kisjeth** [reply]
log_level: INFO
mqtt_host: IP dove è installato mqtt
mqtt_user: nome mqtt
mqtt_pass: password mqtt
port: 1883 (porta default che non ho cambiato)
app:
  - username: account app
    password: password app
    code: hisense-eu (hi smart Life)


Cosa c'è di sbagliato? Perché l'entità è visibile ma non disponibile. Rispetto alla tua configurazione cosa ho cannato?

**Reply by kisjeth** [reply]
"Once everything has been configured, the A/Cs can be blocked from connecting to the internet, as it will no longer be needed. Set them static IP addresses in the router, and write them down.

Note: To avoid the need for manual changes later, make sure the app is aware of the new IP addresses before disconnecting the A/Cs from the internet."


Questo passaggio non mi è chiaro cosa voglia dire.
Ne approfitto per ringraziarti comunque della disponibilità

**Reply by essegi65** [reply]
la porta io ho messo 8888 (ho trovato questa info su un forum) e così mi funziona

**Reply by kisjeth** [reply]
Si, stanotte ho rovistato anche io nel forum e sono giusto alla stessa soluzione. Grazie mille per la tua collaborazione.

**Comment by effe-7** [comment]
ciao a tutti sto provando questa soluzione ma ricevo il seguente errore
" Impossibile salvare la configurazione del componente aggiuntivo, Invalid list for option 'app' in HiSense Air Conditioners (c34eddb3_hisense_ac). Got {'log_level': 'INFO', 'mqtt_host': 'hisense-eu (hi smart Life)', 'mqtt_port': 1883, 'mqtt_user': 'UTENTE-mqtt', 'mqtt_pass': 'PSW-mqtt', 'port': 1883, 'app': {}} "

ho provato anche le porte 8888 - 8883 - 59333
grazie

**Reply by kisjeth** [reply]
Prova ad eliminare il file .json e poi riavviare il componente

**Reply by effe-7** [reply]
risolto grazie..

**Reply by basebuzz** [reply]
Sto cercando di configurare anche io ma non ne vengo a capo, e giorni che ci sto provando, anche io stesso errore. dove elimino il file .json ??
Grazie

**Reply by kisjeth** [reply]
Apri il file editor e cerca la cartella HISENSE. Dentro avrai un file JSON. Eliminalo e riavvia l'integrazione

**Reply by basebuzz** [reply]
grazie 1000. a posto, questa è la mia configurazione funzionante (la documentazione è molto scarsa ed è parecchio faticoso configurare tutto) una volta configurato funziona tutto alla grande! 

log_level: INFO
mqtt_host: 192.168.1.xxx (ip di HA)
mqtt_port: 1883
mqtt_user: *****username MQTT o utente di HA****
mqtt_pass: ******
port: 8888
app:
  - username: ***username app***
    password: ***password app***
    code: hisense-eu

**Comment by effe-7** [comment]
sembra tutto connesso correttamente, in MQTT trovo il dispositivo ma nel registro di connessione al dispositivo ricevo sempre avviso che risulta offline.
"maybe it is offline?"
ma dalla app di hisense lo controllo correttamente.
inoltre appena ho attivato questa connessione mi ha buttato giù melcloud

**Reply by kisjeth** [reply]
Quando a me diceva che potesse essere offline voleva dire che non trovava l'indirizzo IP corretto. L'hai messo statico l'indirizzo del clima?
Dopo che lo metti statico (usando il DHCP del modem) cancelli il file json, riavvii il componente e dovrebbe sistemarsi.
Non ho idea di cosa sia Melcloud.

**Comment by effe-7** [comment]
risolto scusa ... sono rincoglionito in fase di installazione avevo associato il condizionatore alla rete GUEST.
grazie comunque

**Comment by effe-7** [comment]
risolto in parte il problema di connessione... 
ore ricevo questo errore...
Old ID was 10364; new ID is 43959.
I1120 18:56:55.490  web_log.py:206] indirizzoIP-condizionatore [20/Nov/2023:17:56:55 +0000] "POST /local_lan/key_exchange.json HTTP/1.1" 404 213 "-" "-"

**Reply by kisjeth** [reply]
Non ne ho idea. Prova a eliminare tutta l'integrazione, cancella tutta la configurazione relativa e prova a ricrearla
